recipe/enable_language.patch

```diff
@@ -0,0 +1,24 @@
+--- qjson-0.9.0.orig/CMakeLists.txt	2016-11-29 13:44:51.000000000 -0200
```

Comment:
@isuruf I copied from your past patches to fix this.
Do you mind taking a quick look to see if I got it right?